common/app/views/fragments/atoms/atom.scala.html

```diff
@@ -1,15 +1,12 @@
 @import model.content.MediaWrapper
-@import model.content.QandaAtom
-@import model.content.TimelineAtom
-@import model.content.GuideAtom
-@import model.content.ProfileAtom
 @(model: _root_.model.content.Atom, shouldFence: Boolean, amp: Boolean, mediaWrapper: Option[MediaWrapper])(implicit request: RequestHeader, context: _root_.model.ApplicationContext)
+
 @import _root_.model.ShareLinkMeta
-@import _root_.model.content.{InteractiveAtom, MediaAtom, Quiz, StoryQuiz, StoryQuestionsAtom}
+@import _root_.model.content.{InteractiveAtom, MediaAtom, Quiz, StoryQuiz, StoryQuestionsAtom, TimelineAtom, GuideAtom, ProfileAtom, QandaAtom}
 
 @{
     model match {
-        case quiz: Quiz => views.html.fragments.atoms.quiz(quiz, maybeResults = None, showResults = false, ShareLinkMeta(Nil, Nil))
+        case quiz: Quiz => views.html.fragments.atoms.quiz(quiz, maybeResults = None, showResults = false, quiz.shareLinks.getOrElse(ShareLinkMeta(Nil, Nil)))
```

Comment:
Should the template also take an optional rather than a ShareLinkMeta with empty `Seq`? You could also make this the default value of the field passed to `make` and not dealing with optional at all